Update README.md

JuanDiegoGuerra opened the pull request:
Built my first Capstone Project - ( Microverse Gaming Fest )
 -
> Followed project requierements and implemented: 
 - Menu (hamburguer icon) (javascript) ✅
 - Template Structure ✅
 - Different Screen Sizes (css media queries) ✅
 - New theme (gaming fest) 🎨 ✅
 - Responsive Website 📱🖥✅
 - Professional Documentation (readme file) 📄 ✅
 - Project Explanation  [Loom Video](https://www.loom.com/share/3f3011310f8b425cba25deb7a522e535) ✅
 - Website Demo 🚀 (also included in the readme) [Microverse Gaming Fest](https://juandiegoguerra.github.io/Capstone_Project-GamingFest_Page/) ✅

JuanDiegoGuerra committed:
Update README.md

Update all the documentation for this project.

JuanDiegoGuerra committed:
Delete main files to upload the new ones

JuanDiegoGuerra committed:
Create css files

JuanDiegoGuerra committed:
Create index.html

JuanDiegoGuerra committed:
Create app.js

javascript file

Shedrack-Sunday commented on index.html:
```diff
@@ -0,0 +1,229 @@
+                <div class="event-title">
+                    <h1 id="event-title">
+                        MAIN EVENT
+                    </h1>
```
- [ ] There should be only one `h1` element in your project. 
- The reason for having only one `h1` tag in a project is to maintain proper document structure and hierarchical order. The `h1` tag is the most important heading element in HTML and represents the main heading of a page or section. - 
- Having multiple `h1` tags can cause confusion in search engine optimization (SEO) and accessibility, as well as make it difficult for screen readers to understand the main content of the page. It is recommended to use only one h1 tag per page and use other heading tags (h2, h3, etc.) to create a clear and structured document outline.

Shedrack-Sunday commented on index.html:
```diff
@@ -0,0 +1,229 @@
+            </div>
+        </section>
+        <section class="past_events-section">
+            <div class="past_events-main">
```
- [ ] The `About` section of your project should be created in a different folder. this is a major requirement and  i advise you to stick to it 👍 

## Instructions:
![image](https://user-images.githubusercontent.com/50990925/233251101-9e0e31cd-8dbf-4e17-a566-f88bb0894a59.png)

Shedrack-Sunday commented on index.html:
```diff
@@ -0,0 +1,229 @@
+                        <img class="primary-logo" src="img/MicroverseGamingFestLogo.png" width="10%" alt="MicroverseGamingFestLogo">
+                    </a>
+                </div>
+                <ul class="flex-nav-list">
```
- [ ] You missed the black background `nav` that has the social media icons. This might look like a desktop feature that should not be added, but it's actually a design requirement on the site. Kindly add it on both indexes and about page 👍 

## View:
![image](https://user-images.githubusercontent.com/50990925/233251849-89237d44-466d-43fc-bae1-cde6a98af2a8.png)

Shedrack-Sunday commented on index.html:
```diff
@@ -0,0 +1,229 @@
+                </div>
+            </div>
+        </footer>
+    </main> 
```
- [ ] The footer of your project should be different on both indexes and about pages. Right now, you have only one footer. I've made a detailed review of all, that's needed. Let's stick to the requirements, please.

## Your single design:
![image](https://user-images.githubusercontent.com/50990925/233253420-b75f9bc2-b592-4b39-b3a9-c57dca30f2e8.png)

## expected index:
![image](https://user-images.githubusercontent.com/50990925/233253464-12dda348-db2d-473f-8e47-a0826d14b0b0.png)


## Expected about:
![image](https://user-images.githubusercontent.com/50990925/233253525-030a0bdb-6fc4-4d12-b995-9c899cb24577.png)

Shedrack-Sunday commented on index.html:
```diff
@@ -0,0 +1,229 @@
+                        2023.07.07(FRI) ~ 09(SUN)
+                    </h2>
+                    <p id="home-parragraf2">
+                        @ Microverse Gaming Room, Microverse Zoom and more
```
- [ ] Kindly maintain close to identical fonts for this section on both indexes and about sections. 
- For colors, use the one's provided by the design guide

## Design color:
![image](https://user-images.githubusercontent.com/50990925/233254809-1061e5e9-1940-4125-9cef-2c8f9fb77ed5.png)

Shedrack-Sunday commented on index.html:
```diff
@@ -0,0 +1,229 @@
+    </main> 
+    <script src="js/app.js"></script>
+    <!-- <script src="node_modules/bootstrap/dist/js/bootstrap.min.js"></script> -->
+</body>
```
- [ ] After the speakers rendering, there's a `more` button needed to implement. Though the functionality is not compulsory, the button is needed 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50990925/233255840-372a182b-4331-4602-9a82-71564a6ef78e.png)

## Design:
![image](https://user-images.githubusercontent.com/50990925/233255879-37fd1a36-db89-44f7-b051-5f27bc433b04.png)

Shedrack-Sunday reviewed (changes requested):
Hello @JuanDiegoGuerra    👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done well implementing the requirements,  However, there are a few you need to work on, to move to the next project. 💯

 
![](https://user-images.githubusercontent.com/80612925/157397398-00904e26-60ec-46cd-9510-240661c250ef.gif)

## Highlights 
- Nice README structure ✔️ 
- Nice use of HTML 👏 
- Moderate design ✔️ 
  
# Required Changes ♻️

- [ ] In the project requirements, you're meant to highlight a piece of code you're proud of. I could not find it in the video. Kindly see to that 

## Requirements
![image](https://user-images.githubusercontent.com/50990925/233256375-ad218d06-1072-4526-a83a-995ae51ca0e5.png)


_check the comment section for a detailed review_ 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

JuanDiegoGuerra committed:
Create about.html

JuanDiegoGuerra committed:
Update index.html

Added top nav bar that was missing

JuanDiegoGuerra committed:
Create app_about.js

javascript file for "about" html

JuanDiegoGuerra committed:
Update main_styles.css

Updated new elements' styles

JuanDiegoGuerra committed:
Add Media Files

Media files for:
about footer
index footer
"more" button

JuanDiegoGuerra committed:
Update background_elements.css

Updated position of SVG elements

JuanDiegoGuerra committed:
Update README.md

Added the highlight code that I'm proud of.

JuanDiegoGuerra committed:
Fixed Linters

Fixed:
Stylelint linter errors
Eslint linter errors

JuanDiegoGuerra commented:
###  Required changes made ♻
- Added highlight code I'm proud of ✅
- Fixed repetitive h1 tag ✅
- Created About page ✅
- Added top navbar that was missing ✅
- Different footer for Index / About pages ✅
- Used colors and fonts from template ✅
- Implemented "more" button for "featured games" for mobile size ✅

![todolist1](https://user-images.githubusercontent.com/45726631/231703283-083f07c2-f037-4fa4-a834-be7d3bd26e06.gif)

christianbiring1 commented on index.html:
```diff
@@ -9,12 +9,43 @@
 <body>
     <main>
+        <!-- Navbar -->
+<nav class="navbar navbar-expand-lg navbar-dark bg-dark">
```
[OPTIONAL]
- [ ] For a better touch and feel, you may consider increasing the height of your bottom navigation bar and ensure the links are well-centered as you may notice how they aren't.
### Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/233469888-58b94b5d-14cc-4ecd-b8a3-d5a4b34d647e.png)

christianbiring1 commented on index.html:
```diff
@@ -63,9 +94,9 @@ <h2 id="home-title2">
         <section class="main_event-section" id="main_event-section">
```
[OPTIONAL]
- [ ] As the previous reviewer forgot to point that out, You may consider sticking to the original design, I mean the spacing (margin, padding) are supposed to be similar. as you notice how your top section is short and the typo(font-size, weight...) doesn't match with the original template.

### Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/233471012-aeafb6bf-56b2-43f5-b180-f52eb76792a1.png)
### Original template
![image](https://user-images.githubusercontent.com/97100091/233471387-2c8bfd20-b0bd-49c5-90c4-904eaf7addbe.png)

christianbiring1 commented on index.html:
```diff
@@ -136,73 +167,14 @@ <h1 id="event-title">
                 </div>
+                <button class="see-more-btn hide-desktop lato" id="see-more">
+                    MORE <img src="img/svg/arrow.svg" alt="" />
+                </button>
```
- [ ] The `MORE` button here is supposed to be fully functional, the idea is the user must be able to see only a few games at first, when they click the lore button, they can then see the remaining items. We call that `accordion`. Kindly make sure you Implement that.

christianbiring1 commented on css/main_styles.css:
```diff
@@ -27,6 +27,31 @@ html {
 }
 
+/* ------------------ Boostrap Navbara ------------------ */
+.navbar .navbar-nav .nav-link {
```
[OPTIONAL]
- [ ] For a better touch and feel, you may consider adjusting the style on the mobile size menu links, by adjusting the spacing between them as you may notice how messed up they look on your page :point_down: 

![image](https://user-images.githubusercontent.com/97100091/233473813-da3a929c-e4e9-4bba-8343-735e9ab2010a.png)

christianbiring1 commented on about.html:
```diff
@@ -0,0 +1,142 @@
+                </div>
+            </div>
+        </section>
+        <footer class="footer-section" id="footer-section">
```
- [ ] For a better look and feel, You may consider reducing the font size of your element on the footer section as you may notice how big they are.
### Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/233474394-8f7f849f-0bc9-4602-a7f9-a70ea3b581c3.png)

### Original template :point_down: 
![image](https://user-images.githubusercontent.com/97100091/233474497-5ec27556-5743-47a3-8bfc-ac5e2c41a75d.png)

christianbiring1 commented on about.html:
```diff
@@ -0,0 +1,142 @@
+                </div>
+            </nav>
+            </header> 
+        <section class="about_contact-section" id="about_contact-section">
```
[OPTIONAL]
- [ ] Kinldy for a better touch and feel, you may consider adjusting the height of your `sections` both on the index and about page as you may notice how short they appear to be on your page.

christianbiring1 reviewed (changes requested):
Hi @JuanDiegoGuerra , 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!
# STATUS: REQUEST CHANGES :recycle: 
![soclose](https://user-images.githubusercontent.com/97100091/233475306-f1769c2a-9ccf-492b-b0fe-9c2e08b63ca4.gif)

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

JuanDiegoGuerra committed:
Update Footer Section

Modified text size

JuanDiegoGuerra committed:
Add media files

Text clip background

JuanDiegoGuerra committed:
Update index.html

Fixed errors

JuanDiegoGuerra committed:
Update app.js

Refactor main JS in order to implement "MORE" button...

JuanDiegoGuerra committed:
Update background_elements.css

deleted some svg from an specific screen size.

JuanDiegoGuerra committed:
Update main_styles.css

Added new styles to fix some [OPTIONAL] changes.

JuanDiegoGuerra committed:
Fix Linters

Fixed:
Stylint linter errors
Eslint linter errors

JuanDiegoGuerra commented:
### Required changes made ♻
- The "more" button now has to be fully functional and its functionality is now compulsory... ✅
  - "more" button (1st review - _"Though the functionality is not compulsory, the button is needed"_). ❔
  - "more" button (2nd review - _"The MORE button here is supposed to be fully functional"_). ❔
- Reduced the font size of my footer section ✅
- Also Fixed some [OPTIONAL] changes and followed the instructions that says: every comment with the [OPTIONAL] prefix won't stop the approval of this PR.

IngDavidTM reviewed (approved):
Hi @JuanDiegoGuerra  🙋‍♂️, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request follows the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 
<img src="https://media3.giphy.com/media/xT0xezQGU5xCDJuCPe/giphy.gif"/>

## Optional suggestions

- [OPTIONAL] Kindly ensure that the text content within your footer is in a text format rather than being presented as an image. This will allow the text to be more accessible to users who may be using screen readers or have other visual impairments, as well as improve the overall website performance :+1:.

![Screenshot from 2023-04-20 20-00-16](https://user-images.githubusercontent.com/105117832/233516742-159e8e2b-b754-428d-8dc3-bb497d5055a7.png)


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
